src/strategy/json/types/index.ts

```diff
@@ -1 +1,3 @@
+/* istanbul ignore file */
```

Comment:
ts-jest has an issue where interface files are considered in the Istanbul report. This comment makes Istanbul ignore this file